dune.opam

```diff
@@ -32,9 +32,9 @@ depends: [
   "base-threads"
 ]
 conflicts: [
-  "jbuilder" {!= "transition"}
   "odoc" {< "1.3.0"}
   "dune-release" {< "1.3.0"}
+  "jbuilder" {= "transition"}
```

Comment:
Doesn’t this want to be just `jbuilder`? Dune 2 conflicts with *all* versions, right?

Reply:
Nope. Since it doesn't install a `jbuilder` binary, it can be installed alongside the original jbuilder :)